src/test/java/com/atlassian/db/replica/api/AuroraClusterMock.java

```diff
@@ -0,0 +1,105 @@
+package com.atlassian.db.replica.api;
+
+import com.google.common.collect.Sets;
+import org.h2.tools.SimpleResultSet;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Types;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class AuroraClusterMock {
+    private final Connection connection = DriverManager.getConnection("jdbc:h2:mem:testdb", "sa", "");
```

Comment:
why not `IT tests` that run postgress on docker with replica ?

Reply:
Emulating Aurora API is easier/faster without docker.

Reply:
>Emulating Aurora API is easier/faster without docker.